projects/demo-maps/src/app/route-components/route-example-olperformance/services/largelayers.service.ts

```diff
@@ -12,8 +12,8 @@ import {get as getProjection, getTransform} from 'ol/proj';
 import {register} from 'ol/proj/proj4';
 import proj4 from 'proj4';
 import { HttpClient } from '@angular/common/http';
-import { LayersService, VectorLayer, CustomLayer } from '@dlr-eoc/services-layers/src/public-api';
-import { MapOlService } from '@dlr-eoc/map-ol/src/public-api';
+import { LayersService, VectorLayer, CustomLayer } from '@dlr-eoc/services-layers';
+import { MapOlService } from '@dlr-eoc/map-ol';
```

Comment:
That `src/public-api` should not have been here to begin with - thanks a lot for removing it!